spec/rubocop/cop/rspec/void_expect_spec.rb

```diff
@@ -44,4 +44,47 @@
       end
     RUBY
   end
+
+  it 'ignores unrelated method named expect in an example block' do
+    expect_no_offenses(<<~RUBY)
+      it 'something' do
+        MyObject.expect
+      end
+    RUBY
+  end
+
+  it 'flags bare `expect` in example block' do
+    expect_offense(<<~RUBY)
+      it 'something' do
+        expect
+        ^^^^^^ Do not use `expect()` without `.to` or `.not_to`. Chain the methods or remove it.
+      end
+    RUBY
+  end
+
+  context 'when expect has no parent node' do
```

Comment:
I’d just removed all this except the first one.

Reply:
We need the block version below to cover the new guard clause in `on_block` 

I'll delete the rest and get this merged in.